Modules/ITS/src/ITSFhrTask.cxx

```diff
@@ -154,32 +155,54 @@ void ITSFhrTask::initialize(o2::framework::InitContext& /*ctx*/)
         }
       }
     }
-    //define the hitnumber and occupancy array
+    // define the hitnumber and occupancy array
     if (mLayer < NLayerIB) {
       for (int istave = 0; istave < NStaves[mLayer]; istave++) {
-        mHitnumber[istave] = new int[nChipsPerHic[mLayer]];
-        mOccupancy[istave] = new double[nChipsPerHic[mLayer]];
+        mHitnumberLane[istave] = new int[nChipsPerHic[mLayer]];
+        mOccupancyLane[istave] = new double[nChipsPerHic[mLayer]];
         mHitPixelID_InStave[istave] = new std::unordered_map<unsigned int, int>*[nHicPerStave[mLayer]];
         for (int ihic = 0; ihic < nHicPerStave[mLayer]; ihic++) {
           mHitPixelID_InStave[istave][ihic] = new std::unordered_map<unsigned int, int>[nChipsPerHic[mLayer]];
         }
         for (int ichip = 0; ichip < nChipsPerHic[mLayer]; ichip++) {
-          mHitnumber[istave][ichip] = 0;
-          mOccupancy[istave][ichip] = 0;
+          mHitnumberLane[istave][ichip] = 0;
+          mOccupancyLane[istave][ichip] = 0;
         }
       }
     } else {
       for (int istave = 0; istave < NStaves[mLayer]; istave++) {
-        mHitnumber[istave] = new int[nHicPerStave[mLayer]];
-        mOccupancy[istave] = new double[nHicPerStave[mLayer]];
+        mHitnumberLane[istave] = new int[nHicPerStave[mLayer] * 2];
+        mOccupancyLane[istave] = new double[nHicPerStave[mLayer] * 2];
         mHitPixelID_InStave[istave] = new std::unordered_map<unsigned int, int>*[nHicPerStave[mLayer]];
         for (int ihic = 0; ihic < nHicPerStave[mLayer]; ihic++) {
           mHitPixelID_InStave[istave][ihic] = new std::unordered_map<unsigned int, int>[nChipsPerHic[mLayer]];
         }
+        mChipStaveOccupancy[mLayer]->GetYaxis()->SetBinLabel(istave + 1, Form("Stave %i", istave));
+        mChipStaveEventHitCheck[mLayer]->GetYaxis()->SetBinLabel(istave + 1, Form("Stave %i", istave));
+	if (mLayer < 5) {
         for (int ihic = 0; ihic < nHicPerStave[mLayer]; ihic++) {
-          mHitnumber[istave][ihic] = 0;
-          mOccupancy[istave][ihic] = 0;
+          mHitnumberLane[istave][2 * ihic] = 0;
+          mHitnumberLane[istave][2 * ihic + 1] = 0;
+          mOccupancyLane[istave][2 * ihic] = 0;
+          mOccupancyLane[istave][2 * ihic + 1] = 0;
+          mChipStaveOccupancy[mLayer]->GetXaxis()->SetBinLabel(2*ihic , Form("%s", OBLabel34[2*ihic]));
+          mChipStaveOccupancy[mLayer]->GetXaxis()->SetBinLabel(2*ihic + 1 , Form("%s", OBLabel34[2*ihic + 1]));
```

Comment:
2*ihic+2 here for the same reason (only the bin number!)

Reply:
Please check the labels are correctly displayed also (as you can see in your QC GUI, the last bin is missing a label because of that)